main/adapter/adapter.h

```diff
@@ -140,6 +141,24 @@ enum {
     PAD_RJ,
 };
 
+enum {
```

Comment:
this enum was added so I could minimize the number of characters needed to represent the 15 buttons of the controller.  With this new enum only 5 characters are needed.